auto_nag/scripts/defectenhancementtask.py

```diff
@@ -66,30 +66,41 @@ def get_bugs(self, date='today', bug_ids=[]):
         if len(bugs) == 0:
             return {}
 
+        # Apply inverse transformation to get the type name from the encoded labels.
+        labels = self.model.clf._le.inverse_transform([0, 1, 2])
+        labels_map = {label: index for label, index in zip(labels, [0, 1, 2])}
+
         # Get the encoded type.
         indexes = probs.argmax(axis=-1)
-        # Apply inverse transformation to get the type name from the encoded value.
-        suggestions = self.model.clf._le.inverse_transform(indexes)
 
         results = {}
-        for bug, prob, index, suggestion in zip(bugs, probs, indexes, suggestions):
+        for bug, prob, index in zip(bugs, probs, indexes):
+            suggestion = labels[index]
             assert suggestion in {'defect', 'enhancement', 'task'}, f'Suggestion {suggestion} is invalid'  # noqa
 
             if bug['type'] == suggestion:
                 continue
 
+            defect_prob = prob[labels_map['defect']]
+            enhancement_prob = prob[labels_map['enhancement']]
+            task_prob = prob[labels_map['task']]
+
+            def nice_round(val):
+                return int(round(100 * val))
+
             results[bug['id']] = {
                 'id': bug['id'],
                 'summary': self.get_summary(bug),
                 'type': bug['type'],
                 'bugbug_type': suggestion,
-                'confidence': int(round(100 * prob[index])),
+                'confidence': nice_round(prob[index]),
```

Comment:
nit: you don't use it anymore in the template: so remove it

Reply:
I'm using it for sorting

Reply:
Ok, so let it like this.